control-plane/plugin/src/resources/mod.rs

```diff
@@ -104,10 +104,22 @@ pub enum DrainResources {
 /// The types of resources that support the 'label' operation.
 #[derive(clap::Subcommand, Debug)]
 pub enum LabelResources {
-    /// Label node with the given ID.
+    /// Adds or removes a label from the specified node.
     Node {
+        /// The id of the node to label.
         id: NodeId,
+        /// The label to be added or removed from the node.
+        /// To add a label, specify the following format:
+        /// ${key}=${value}
+        /// To remove a label, specify the following format:
+        /// ${key}-
+        /// A label key and value must begin with a letter or number, and may contain letters,
```

Comment:
So key beginning with a number is a valid key?

Reply:
Yes it is